tilequeue/query/postgres.py

```diff
@@ -67,9 +64,15 @@ def __init__(self, sources, query_generator):
     def __call__(self, zoom, bounds):
         queries = []
         for source in self.sources:
-            if source.start_zoom <= zoom:
-                query = self.query_generator(source.template, bounds, zoom)
-                queries.append(query)
+            template_queries = []
+            for template_spec in source.template_specs:
+                if template_spec.zoom_start <= zoom < template_spec.zoom_stop:
```

Comment:
Should this be `zoom_start <= zoom <= zoom_stop`? In other words, is the start to end range of zooms inclusive of the stop or exclusive?